### PR TITLE
Bug 1193462 - Sign in from tour/remote tabs now takes you directly to signin page

### PR DESCRIPTION
**Pull request by sleroux**
Note: I needed to add a new string for the intro view's accessibilityLabel as it didn't exist before.

### CODE REVIEW

File: Client/Frontend/Browser/BrowserViewController.swift [hunk 1]
```diff
@@ -1846,14 +1847,22 @@ extension BrowserViewController: IntroViewControllerDelegate {
     }
 
     func presentSignInViewController() {
-        // TODO When bug 1161151 has been resolved we can jump directly to the sign in screen
-        let settingsNavigationController = SettingsNavigationController()
-        settingsNavigationController.profile = self.profile
-        settingsNavigationController.tabManager = self.tabManager
+        let signInVC = FxAContentViewController()
+        signInVC.delegate = self
```
Comment (thebnich) [inline Client/Frontend/Browser/BrowserViewController.swift]: I noticed `FxAContentViewController` `delegate` is not `weak`. Can we fix that while here?

File: Client/Frontend/Browser/BrowserViewController.swift [hunk 2]
```diff
@@ -1861,6 +1870,29 @@ extension BrowserViewController: IntroViewControllerDelegate {
     }
 }
 
+extension BrowserViewController: FxAContentViewControllerDelegate {
+    func contentViewControllerDidSignIn(viewController: FxAContentViewController, data: JSON) -> Void {
+        if data["keyFetchToken"].asString == nil || data["unwrapBKey"].asString == nil {
+            // The /settings endpoint sends a partial "login"; ignore it entirely.
+            NSLog("Ignoring didSignIn with keyFetchToken or unwrapBKey missing.")
```
Comment (rnewman) [inline Client/Frontend/Browser/BrowserViewController.swift]: `log.debug`?

File: Client/Frontend/Browser/BrowserViewController.swift (hunk 1 above)
Comment (sleroux) [inline Client/Frontend/Browser/BrowserViewController.swift]: :+1:

**Comment by rnewman** [comment]
LGTM with that one change.

**Comment by rnewman** [comment]
We should probably not land this until v1.0 is approved.

**Comment by sleroux** [comment]
Sounds good. I've added the hasAccount change. We'll wait for 1.0.1 since this adds a new string.

**Comment by rnewman** [comment]
This'll be safe to merge if we don't mind a little pain if we have to resubmit.

**Comment by sleroux** [comment]
Now that it's approved, I'll merge this down. I'll also merge down a copy into our v1.0 branch.

**Pull request merged**